writeup/conStruct.tex

```diff
@@ -277,7 +277,8 @@ \section*{Abstract}
 sample from a set of two-dimensional population layers, 
 and, within each layer, estimates a rate at which relatedness decays with distance.
 This thereby explicitly addresses the ``clines versus clusters" problem in 
-modeling population genetic variation.
+modeling population genetic variation,
+and avoids problems with overfitting that nonspatial models are prone to.
```

Comment:
I'm not thinking of the elegant way to say this, but Reviewer 1 suggests highlighting this, and I agree it's a good idea.